src/python_testing/TC_EPM_2_2.py

```diff
@@ -0,0 +1,109 @@
+            TestStep("4c", "TH reads from the DUT the Voltage attribute. Verify the read is successful and that the value is between 229'000 and 231'000 mV. Note the value read."),
+            TestStep("5", "Wait 3 seconds"),
+            TestStep("5a", "TH reads from the DUT the ActivePower attribute. Verify the read is successful, that the value is between 0'980'000 and 1'020'000 mW, and the value is different from the value read in step 4a."),
+            TestStep("5b", "TH reads from the DUT the ActiveCurrent attribute. Verify the read is successful, that the value is between 3'848 and 4'848 mA, and the value is different from the value read in step 4b."),
+            TestStep("5c", "TH reads from the DUT the Voltage attribute. Verify the read is successful, that the value is between 229'000 and 231'000 mV, and the value is different from the value read in step 4c."),
+            TestStep("6", "TH sends TestEventTrigger command to General Diagnostics Cluster on Endpoint 0 with EnableKey field set to PIXIT.EPM.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to PIXIT.EPM.TEST_EVENT_TRIGGER for Stop Fake Readings Test Event."),
+        ]
+
+        return steps
+
+    @async_test_body
+    async def test_TC_EPM_2_2(self):
+
+        self.step("1")
+        # Commission DUT - already done
+
+        self.step("2")
+        await self.check_test_event_triggers_enabled()
+
+        self.step("3")
+        await self.send_test_event_trigger_start_fake_1kw_load_2s()
+
+        # After 3 seconds...
+        self.step("4")
+        time.sleep(3)
+
+        self.step("4a")
+        active_power = await self.check_epm_attribute_in_range("ActivePower", 980000, 1020000)  # 1kW +/- 20W
+
+        self.step("4b")
+        active_current = await self.check_epm_attribute_in_range("ActiveCurrent", 3848, 4848)    # 4.348 A +/- 500mA
+
+        self.step("4c")
+        voltage = await self.check_epm_attribute_in_range("Voltage", 229000, 231000)      # 230V +/- 1V
+
+        self.step("5")
+        # After 3 seconds...
+        time.sleep(3)
+
+        self.step("5a")
+        active_power2 = await self.check_epm_attribute_in_range("ActivePower", 980000, 1020000)  # 1kW +/- 20W
+
+        self.step("5b")
+        active_current2 = await self.check_epm_attribute_in_range("ActiveCurrent", 3848, 4848)    # 4.348 A +/- 500mA
+
+        self.step("5c")
+        voltage2 = await self.check_epm_attribute_in_range("Voltage", 229000, 231000)      # 230V +/- 1V
+
+        asserts.assert_not_equal(active_power, active_power2,
+                                 f"Expected ActivePower readings to have changed {active_power}, {active_power2}")
+        asserts.assert_not_equal(active_current, active_current2,
+                                 f"Expected ActiveCurrent readings to have changed {active_current}, {active_current2}")
```

Comment:
Move asserts to immediately below the reads so that they are in the correct test steps

Reply:
Done